### ISSUE TITLE
token update

### ISSUE THREAD

**Issue by zebiden** [schwifty42069/pmg#8]
morning Boss i me doing something wrong.the token keep changing with in 2 second ,it will not give chance to the channel to open. it look like impossible to get the token in 2 second open in the channel in the same time. what you this Boss. thank you

**Comment by zebiden** [comment]
hey brother is it working. i am using vlc i get the token update but it seem the token changing every second in the channels.can you please help me.thank you

**Comment by schwifty42069** [comment]
Unfortunately, there is  a bug in VLC that is preventing the m3u generated by the script to work with it. The host is now blocking the VLC http user-agent header, which is why the #EXTVLCOPT:http-user-agent line was added to the m3u generated by the script. This changes the header, but for some reason VLC only retains the header for the initial playlist call, as soon as it calls the chunks from the playlist, it returns to the default "VLC <version>" header, and thus is being blocked by the host.

My recommendation is to download MPV. It works on both Windows and Linux and plays the m3u generated by the script flawlessly. (:

I'm gonna close this, if you still have a problem getting the streams to play with MPV, open a new issue.
